docs/source/intervention_models/crm_mgmt/vivarium_polypill.rst

```diff
@@ -107,6 +112,18 @@ those who are not receiving the intervention. The the new adherence rates for si
 Please note that the intervention works on the overall level of adherence (% adherent in the table above). 
 The percents for primary and secondary nonadherence are now arbitrary.  
 
+**Blood Pressure Treatment Assignment**
+
+Any simulant on 'three drugs at half dose' or lower on the ladder will be 
+assigned to 'three drugs at half dose' when they are put on the polypill intervention. 
+Use the efficacy normally assigned for this regimen. 
+
+Any simulant at 'three drugs at standard dose' on the ladder will continue 
+with the regimen they have while on the polypill intervention. 
+
+Simulants on the polypill intervention can move up the ladder to three 
+drugs at standard dose based on the usual treatment ramp shown :ref:`here <us_cvd_concept_model>`. 
```

Comment:
Are there ladder rungs between three drugs at half dose and three drugs at full dose?  If so, I think polypill should skip them.

Reply:
Nope! These are next to each other on the ladder.